openwisp_monitoring/db/backends/__init__.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from importlib import import_module
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import DatabaseError
+
+logger = logging.getLogger(__name__)
+
+TIMESERIES_DB = getattr(settings, 'TIMESERIES_DATABASE', None)
+if not TIMESERIES_DB:
+    TIMESERIES_DB = {
+        'BACKEND': 'openwisp_monitoring.db.backends.influxdb',
+        'USER': 'openwisp',
+        'PASSWORD': 'openwisp',
+        'NAME': 'openwisp2',
+        'HOST': 'localhost',
```

Comment:
please let's keep backward compatibility with the settings used before, this will facilitate the upgrade.

Eg:

```python
{
# ...
    'USER': getattr(settings, 'INFLUXDB_USER', 'openwisp')
# ...
}
```